elyra/pipeline/kfp/processor_kfp.py

```diff
@@ -450,6 +449,7 @@ def _cc_pipeline(self,
         # and its parent's outputs.
 
         PipelineProcessor._propagate_operation_inputs_outputs(pipeline, sorted_operations)
+        component_task_factories = {}
```

Comment:
Is there a reason why we aren't just using `target_ops` for this?  They're both have the same keys and values. 

I do see that `target_ops` tends to be exclusive to generic components while `component_task_factories` appears to be exclusive to custom components, but since operation ids (i.e., the keys to these dictionaries) will be unique across those two sets, I'm not sure `component_task_factories` is necessary.

If we do retain c-t-f, could you please add some comments surrounding its use?  I don't really follow the 'task' or 'factory' portion of its name.